cg/cli/demultiplex/finish.py

```diff
@@ -26,16 +22,10 @@ def finish_group():
 @click.pass_obj
 def finish_all_cmd(context: CGConfig, bcl_converter: str, dry_run: bool) -> None:
```

Comment:
The `bcl_converter` is no longer used, but removing it would break our automation scripts. Should we do it now or later in a separate PR?

Reply:
Same for all the commands below

Reply:
Better to split it up